canonical/ubuntu.com
# WD-8567 - update text in /data-privacy

## Done

Add text to /data-privacy

## QA
- [demo link](https://ubuntu-com-13568.demos.haus/legal/data-privacy)
- [copy doc](https://docs.google.com/document/d/1AlOE2_-BqZNP4hiiuf8kZney8g1zvqAlDYl2u3X6m6I/edit)
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- Run through the following [QA steps](https://discourse.canonical.com/t/qa-steps/152)
- Check that the page is correctly updated according to copy doc

## Issue / Card
[WD-8765](https://warthogs.atlassian.net/browse/WD-8765)

Fixes #

## Screenshots

[WD-8765]: https://warthogs.atlassian.net/browse/WD-8765?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
